File: C++/Answer8.cpp
#include<iostream>
using namespace std;

int main () {
    int alpha = 42;
    int beta = 84;
    std::cout << "A : "<< alpha << std::endl;
    std::cout << "B : "<< beta << std::endl;
    alpha = alpha + beta;
    beta = alpha - beta;
    alpha = alpha - beta;
    std::cout << "A : "<< alpha << std::endl;
    std::cout << "B : "<< beta << std::endl;
    return 0;
}

File: Java/Answer8.java
class Answer8 {
    public static void  main(String[] args) {
        int alpha = 235;
        int beta = 633;
        System.out.println("A : "+ alpha);
        System.out.println("B : "+ beta);
        alpha = alpha + beta;
        beta = alpha - beta;
        alpha = alpha - beta;
        System.out.println("A : "+ alpha);
        System.out.println("B : "+ beta);
    }
}